src/main/java/cms/gov/madie/measure/SecurityConfig.java

```diff
@@ -20,6 +20,12 @@ protected void configure(HttpSecurity http) throws Exception {
         .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
         .and()
         .oauth2ResourceServer()
-        .jwt();
+        .jwt()
+        .and()
+        .and()
```

Comment:
Extra `and()` possibly.

Reply:
@jkotanchik-SB actually no, it took me a while to find it out otherwise I would get error: The method headers() is undefined for the type OAuth2ResourceServerConfigurer<HttpSecurity>

Reply:
Thanks @sb-cecilialiu for the clarification. That's rather weird.